src/confluent_kafka/schema_registry/_sync/schema_registry_client.py

```diff
@@ -945,6 +1012,55 @@ def get_version(
 
         return registered_schema
 
+    def get_version_schema_string(
+        self, subject_name: str, version: Union[int, str] = "latest",
+        deleted: bool = False, fmt: Optional[str] = None
+    ) -> str:
+        """
+        Retrieves a specific schema registered under ``subject_name`` and ``version``.
+        Only the unescaped schema string is returned.
+
+        Args:
+            subject_name (str): Subject name.
+            version (int): version number. Defaults to latest version.
+            deleted (bool): Whether to include deleted schemas.
+            fmt (str): Format of the schema.
+
+        Returns:
+            str: Schema string for this version.
+
+        Raises:
+            SchemaRegistryError: if the version can't be found or is invalid.
+
+        See Also:
+            `GET Subject Versions API Reference <https://docs.confluent.io/current/schema-registry/develop/api.html#get--subjects-(string-%20subject)-versions-(versionId-%20version)-schema>`_
+        """  # noqa: E501
+
+        query = {'deleted': deleted, 'format': fmt} if fmt is not None else {'deleted': deleted}
```

Comment:
Same here on deleted not being in docs reference, are there other routes we need to add deleted to that have it missing?